### PR TITLE
finish markup for header and problem sections, add a missing svg

### PR DESCRIPTION
**Pull request by prechter**
Adding markup for these two sections I'm calling "header" and "problem" sections. This PR is maybe a bit light (still trying to find the right balance). Hopefully it's quick for someone to approve or make comments on. Added an SVG I'm going to use as the background image for the empty `<div class="pill-bottles"></div>`

screen shot 2019-01-25 at 5 01 50 pm

### CODE REVIEW

File: src/images/Pill_Bottles.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....933v25.909h2.641zM182.775 204.883v-25.898a1.316 1.316 0 0 0-1.32-1.331c-.349.006-.68.149-.923.399a1.314 1.314 0 0 0-.375.932v25.898h2.618zM206.97 204.883v-25.898a1.319 1.319 0 0 0-1.321-1.331 1.32 1.32 0 0 0-1.321 1.32v25.909h2.642zM231.164 204.883v-25.898a1.3 1.3 0 0 0-.383-.94 1.317 1.317 0 0 0-1.871-.004 1.317 1.317 0 0 0-.387.933v25.909h2.641zM255.358 204.883v-25.898a1.316 1.316 0 0 0-1.32-1.331 1.32 1.32 0 0 0-1.321 1.32v25.909h2.641zM279.553 204.883v-25.898a1.3 1.3 0 0 0-.383-.94 1.319 1.319 0 0 0-.938-.391 1.319 1.319 0 0 0-1.298 1.331v25.898h2.619zM303.747 204.883v-25.898a1.332 1.332 0 0 0-.812-1.229 1.324 1.324 0 0 0-1.442.285 1.317 1.317 0 0 0-.387.933v25.909h2.641z" fill="#C0D5D0"/><path d="M146.797 190.273h163.395a.797.797 0 0 1 .797.796v9.342a5.01 5.01 0 0 1-5.01 5.007H151.01a5.006 5.006 0 0 1-5.01-5.007v-9.33a.805.805 0 0 1 .489-.746.795.795 0 0 1 .308-.062z" fill="#F8F6E8"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h202v358H0z"/></clipPath><clipPath id="clip1"><path fill="#fff" transform="translate(258 88)" d="M0 0h111v244H0z"/></clipPath><clipPath id="clip2"><path fill="#fff" transform="translate(146 164)" d="M0 0h165v227H0z"/></clipPath></defs></svg>
```
Comment (olets) [inline src/images/Pill_Bottles.svg:1]: SVG might be the best option here, but if you're going to use it as an `img`'s `src` you can probably choose whatever format gives you the smallest file. This is 5.5K. With SVGO (using the GUI SVGOMG https://jakearchibald.github.io/svgomg/, with everything except "Remove xmlns" and "Remove viewBox" turned on, and Precision at 1) I can get this down to 4k, but I can get a 2x (retina-sized) png down to 3.6K, and a 1x png down to 1.9K.

To do that I dropped the SVG into Figma http://figma.com, exported as PNG, ran the PNG through ImageAlpha https://pngmini.com/ (reduced to 16 colors), and then through ImageOptim https://imageoptim.com/mac (if you have ImageOptim installed, ImageAlpha will give you the option of automatically using it, so those two apps are really just one step)

**Review by olets — APPROVED** [review]
lgtm 🦅

There's no "right" and "wrong" here, but I would have broken this up into a couple single-focus commits. Maybe one for the header nav, one for the problem section content, and one for the pill svg. In this case the commit message really was able to describe the whole thing, so there's def no "problem" with what you did. But I find the "each commit is one cohesive thing" mindset helpful — it forces me to think of the work as a bunch of discrete items, and as the work gets more and more complex that approach keeps the commit history a helpful, communicative log — 

```
finishes the top of the homepage
```
vs
```
homepage text styles
homepage mobile text styles
homepage hero
homepage hero lazy loads
```
etc

Especially for internal PRs like this, where iterating is an option, my personal approach is:
- everything in the PR holds together as some cohesive thing
- everything I'm doing in this iteration of that thing is included in the PR
- the thing isn't so huge or complex that a reviewer, who's reading from top to bottom, won't understand it (want the reviewer to notice the bigger picture stuff like "I'd structure it differently" not just be overwhelmed and reduced to looking for glaring typos etc)

File: src/html/partials/header.html [hunk 2]
```diff
@@ -1,8 +1,18 @@
 <header>
+
   <div class="wrapper--title">
     <h1 class="header__title">A Viget Exploration</h1>
   </div>
-  <div>
-    <img src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
-  </div>
+
+  <img class="header__img" src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
```
Comment (jeremyfrank) [inline src/html/partials/header.html:7]: If you have `header__img` here, might want to have `header` above, e.g. `<header class="header">`, but you can always address that when you get to the css/styling phase.

File: src/html/partials/header.html [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+
+  <nav role="navigation" class="nav">
+    <ul class="nav__list">
+      <li>The Problem</li>
+      <li>Key Factors</li>
+      <li>The Question</li>
+      <li>The Solution</li>
```
Comment (jeremyfrank) [inline src/html/partials/header.html]: For accessibility, these would need to be in page anchors, e.g.
```
<li><a href="#solution">The Solution</a></li>
```
and that would link down to "the solution" section (id attribute) further down the page...
```
<section id="solution">
  ...
</section>
```

**Review by jeremyfrank — APPROVED** [review]
Looks good. Just one small update with the pillbar links.

File: src/images/Pill_Bottles.svg (hunk 1 above)
Comment (prechter) [reply src/images/Pill_Bottles.svg:1]: Thanks for all the super helpful feedback! I did drop all the SVGs through SVGO specifically keeping the viewBox per Jeremy's suggestion. Although, knowing when to use PNG vs SVG etc is an area I'm just now gaining some understanding, so I'll be more aware of this going forward.

**Pull request merged**